modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/model/preferences/WorkspacePreferences.java

```diff
@@ -41,6 +41,14 @@ public Map<String, EmailReportPreferences> addEmailReportPreference(
 		return _emailReportPreferences;
 	}
 
+	public Map<String, EmailReportPreferences> deleteEmailReportPreference(
```

Comment:
let's make it plural `deleteEmailReportPreferences`

Reply:
Renamed to `removeEmailReportPreference` to match `removeIndividualSegmentPreference`